plasma_framework/test/src/exits/payment/PaymentInFlightExitModelUtils.test.js

```diff
@@ -3,22 +3,226 @@
 
 const PaymentInFlightExitModelUtils = artifacts.require('PaymentInFlightExitModelUtilsMock');
 
-const { expectRevert, time } = require('openzeppelin-test-helpers');
+const { expectRevert, time, constants } = require('openzeppelin-test-helpers');
 const { expect } = require('chai');
 
+const { EMPTY_BYTES_32 } = require('../../../helpers/constants.js');
+
 contract('PaymentInFlightExitModelUtils', () => {
     const MAX_INPUTS = 4;
     const MAX_OUTPUTS = 4;
     const MIN_EXIT_PERIOD = 1000;
     const ALL_INPUTS_PIGGYBACKED = 2 ** 0 | 2 ** 1 | 2 ** 2 | 2 ** 3;
     const ALL_OUTPUTS_PIGGYBACKED = 2 ** MAX_INPUTS | 2 ** (MAX_INPUTS + 1)
-    | 2 ** (MAX_INPUTS + 2) | 2 ** 2 ** (MAX_INPUTS + 2);
+    | 2 ** (MAX_INPUTS + 2) | 2 ** (MAX_INPUTS + 3);
+
+    describe('isInputEmpty', () => {
+        it('returns true when the input is empty', async () => {
+            const modelUtils = await PaymentInFlightExitModelUtils.new(0, 0);
+
+            for (let i = 0; i < MAX_INPUTS; i++) {
+                const actual = await modelUtils.isInputEmpty(i);
+                expect(actual).to.be.true;
+            }
+        });
+
+        it('returns false when the outputId is not empty', async () => {
+            const modelUtils = await PaymentInFlightExitModelUtils.new(0, 0);
+
+            const dataWithNonEmptyOutputId = {
```

Comment:
I would make a constant `emptyWithdrawData` on a higher scope and in each test I would just update one value in it. That would save some lines of code.